04/Components/App.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { v4 as uuid } from "uuid";
+
+import File from "./File";
+import List from "./List";
+
+
+export default class App extends React.Component {
+    fileRef = React.createRef();
+
+    state = {
+        filesList: [],
+    }
+
+    handleFileLoad = () => {
+        const files = this.fileRef.current.getFiles();
+
+        for (const file of files) {
+            if (file && file.type.includes('text/plain')) {
+                this.readFile(file);
+            } else {
+                alert(`${file.name} has wrong file type!`);
+            }
+        }
+    }
+
+    readFile(file) {
+        let reader = new FileReader();
+
+        reader.onload = () => {
+            const { name: fileName, size: fileSize } = file;
+            const fileContent = reader.result;
+            const fileObject = { fileName, fileSize, fileContent };
+
+            this.setFilesListState(fileObject);
+        }
+
+        reader.readAsText(file);
+    }
+
+    setFilesListState(fileData) {
```

Comment:
`set` oznacza "ustaw" - dla mnie to nadpisanie poprzedniej wartości. Jeśli dodajemy kolejną (a tutaj tak robisz) to ja bym zaczął od `add` lub `insert`. Niby mała rzecz, ale mi to ułatwiłoby życie :D

Reply:
Ach, racja – dzięki za przypomnienie :).